game.js

```diff
@@ -0,0 +1,143 @@
+var sizeOfOneSquare = 40, sizeOfFoundation = 3;
+var canvas = document.getElementById("main"), context = 
+canvas.getContext("2d");
+function drawmap() {
+    for (var i=0; i<sizeOfFoundation; i++){
+        for (var j=0; j<sizeOfFoundation; j++){
+            context.strokeRect(0+i*sizeOfOneSquare, 0+j*sizeOfOneSquare, 
+sizeOfOneSquare, sizeOfOneSquare);
+        }
+    }
+}
+drawmap();
+var WidthOfWindow =  window.innerWidth*0.45, HeightOfWindow = 
+window.innerHeight*0.2;
+var flagForChangeStep = true, flagWinner = false;
+var arrayForO = [], arrayForX = [], countSteps = 0;
+
+function showCoords(evt) {
+    if (evt.clientX>=WidthOfWindow && evt.clientX 
```

Comment:
ANYA. How many times I should write you use <== and >== and USE ONLY three character of comparing ???????????????????///

Reply:
in js absent such operators, I search it because visual emphasized it

Reply:
do you want to say that js doesn't have operators like >== and <== ??? Do not make me upset with you.